.github/workflows/ci.yml

```
@@ -66,7 +66,7 @@ jobs:
       - run: npm ci
       - run: npm run build
       - run: npm run test:unit
-      - run: npm run test:types -- --target 4.5,5.0,current
+      - run: npm run test:types -- --target 5.0,current
```

Nice! To draw your attention, `tstyche@3.1.1` and above makes it possible to test agains a range of versions:

```suggestion
      - run: npm run test:types -- --target '>=5.0'
```

---

Nice, but doing that the tests ran 5 times slower. 
I do not know if it's worth it for now (but I do keep that in mind)